bedrock/firefox/templates/firefox/os/index.html

```diff
@@ -77,220 +55,81 @@
   <main role="main">
     <section class="fxos-hero-space">
       <div class="content-wrapper">
-      {% block hero_copy %}
-        <div class="phone">
-          <div class="fxos-cta">
-            <h1><img src="{{ static('img/firefox/os-new/fxos-wordmark.svg') }}" width="420" alt="Firefox OS" data-png="{{ static('img/firefox/os-new/fxos-wordmark.png') }}" data-fallback="true"></h1>
-            <p class="tagline">{{_('Designed to give you just what you need, just when you need it.')}}</p>
-
-            <div class="primary-cta-signup signup hidden">
-              <a href="#email-form-wrapper" class="cta-button get-newsletter newsletter-signup-toggle" role="button" aria-haspopup="true">{{_('Sign me up')}}</a>
-              <p class="signup-txt">{{_('Sign up for our monthly newsletter and get the latest on Firefox OS')}}</p>
-            </div>
-            <a href="#get-device" class="primary-cta-phone cta-button get-phone hidden" role="button" aria-haspopup="true">{{_('Get a phone')}}</a>
+        <div class="hero-content">
+          <h1>{{ _('Experience Firefox OS on your Android device')}}</h1>
+          <p>{{_('Introducing Firefox OS 2.5 Developer Preview, an experimental app that lets you use Firefox OS on your Android device.')}}</p>
+          <div class="primary-cta">
+            <a href="https://d2yw7jilxa8093.cloudfront.net/B2GDroid-mozilla-central-nightly-latest.apk" class="cta-button" data-pixel="52161" data-id="Mozilla_B2G_Activity_Tag_1">{{_('Get the Android App')}}</a>
```

Comment:
Not sure how long-term this link is (the cloudfront URL seems random), but if it's going to be changing with some frequency, should consider moving to settings.